queries/groovy/folds.scm

```diff
@@ -0,0 +1,4 @@
+[
+  (closure)
+  (argument_list)
+] @fold
```

Comment:
What about lists, maps, and blocks?

Reply:
All blocks are closures in my grammar. will throw in lists and maps